Scripts/Services/LootGeneration/RunicReforging/RunicReforging.cs

```diff
@@ -1977,12 +1965,13 @@ public static void GenerateRandomItem(Item item, Mobile killer, int basebudget,
                     if (forcedsuffix == ReforgedSuffix.None && budget >= Utility.Random(2700))
                         suffix = ChooseRandomSuffix(item, budget, prefix);
 
+
                     if (!IsPowerful(budget))
                     {
                         mods = Math.Max(1, GetProperties(5));
 
-                        perchigh = Math.Max(50, Math.Min(500, budget) / mods);
-                        perclow = Math.Max(20, perchigh / 3);
+                        perchigh = Math.Max(50, Math.Min(550, budget) / mods);
+                        perclow = Math.Max(20, Convert.ToInt32(perchigh / 3 * perc));
```

Comment:
the perclow was always ignored, so I've scaled it down based on the perc rolled at the start.
Higher luck will give higher minimums, but will always be a maximum of 33% of the perchigh if perc was 1.0, if perc rolled 0% then you would still roll 0 to perchigh as previously. But luck always matters now.